abuild.in

```diff
@@ -765,6 +765,9 @@ pre_split() {
 		return 0
 	fi
 	# the subpackages should not inherit these from main package
+	origdepends="$depends"
+	origindepends="$origdepends" # temporary transitioning variable for aports
```

Comment:
why both `origdepends` and `origindepends` and what is the difference between those?

Reply:
The idea is to have an easy way to distinguish temporary uses from those intended to be permanent while cleaning up how aports define depends in split functions.

For example, change `depends="$depends ..."` to `depends="$origindepends ..."` in an automatic operation then switch to `$origdepends` after review if no better approach was appropriate.